performance/numerical/src/nativeInterop/cinterop/pi.def

```diff
@@ -0,0 +1 @@
+package = pi
```

Comment:
What is the point of that?

Reply:
It was just blindly copied from my old behchmark.
What should be here?